share/pkgs/sudoers/opennebula-node-firecracker

```diff
@@ -0,0 +1 @@
+oneadmin ALL=(ALL:ALL) NOPASSWD: ONE_FIRECRACKER
```

Comment:
Is this package going to be installed alongside with other package (e.g., node-kvm?).
If not, probably you'll need some networking command lists (e.g., ONE_NET, ONE_OVS).

Reply:
For the moment the idea is to install it along with KVM packages as it also required KVM.